tests/api-mongo.test.js

```diff
@@ -466,6 +466,30 @@ describe('MongoDB Words', () => {
         });
     });
 
+    it('should return a list of igbo terms when using english by using single qoutes', (done) => {
+      const keyword = '\'water\'';
+      getWords({ keyword })
+        .end((_, res) => {
+          expect(res.status).to.be.equal(200);
+          expect(res.body).to.be.an('array');
+          expect(res.body).to.have.lengthOf.at.least(5);
+          expect(every(res.body, (word) => word.includes('water'))).to.equal(true);
+          done();
+        });
+    });
+
+    it('should also return a list of igbo terms when using english by using double qoutes', (done) => {
+      const keyword = '"water"';
+      getWords({ keyword })
+        .end((_, res) => {
+          expect(res.status).to.be.equal(200);
+          expect(res.body).to.be.an('array');
+          expect(res.body).to.have.lengthOf.at.least(5);
```

Comment:
Also, where did the number `5` come from? Why did you go with at least the length of `5`?